packages/liveblocks-core/src/api-client.ts

```diff
@@ -982,6 +988,19 @@ export function createApiClient<M extends BaseMetadata>({
   /* -------------------------------------------------------------------------------------------------
    * Notifications (Room level)
    * -----------------------------------------------------------------------------------------------*/
+  async function getRoomUnreadInboxNotificationsCount(options: {
+    roomId: string;
+  }) {
+    const { count } = await httpClient.get<{ count: number }>(
+      url`/v2/c/rooms/${options.roomId}/inbox-notifications/count`,
+      await authManager.getAuthValue({
+        requestedScope: "comments:read",
+        roomId: options.roomId,
+      })
```

Comment:
@ofoucherot @marcbouchenoire I am not very well versed in the current auth manager but we seem to request cached token for this very scope `comments:read` for a lot of other methods (`createThread`, `deleteThread`) currently. Should we be requesting `room:read` scope in this particular method?

Reply:
Yes, for inbox-notifications it should be `room:read`.

Reply:
I agree that `room:read` is the right one.

That said, if you use comments without storage, you'd still want to have the notifications. In which case you need the less powerful permission (`comments:read`) to still provide notifications.

Wondering if this is a missing piece in our system. And we should add a `notifications:write/read` permission.